assembl/static2/yarn.lock

```diff
@@ -5,12 +5,10 @@
 "@2fd/command@^1.1.2":
   version "1.1.2"
   resolved "https://registry.yarnpkg.com/@2fd/command/-/command-1.1.2.tgz#b831d511bdd3b4e2d88139aaddb06d3d6aa0b08a"
-  integrity sha1-uDHVEb3TtOLYgTmq3bBtPWqgsIo=
```

Comment:
this is strange. Your `yarn` may be out of date. Try to update it, rebase develop, re-run `yarn` and verify that it is fixed